build.gradle

```diff
@@ -25,9 +25,14 @@ dependencies {
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
         exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
     }
-    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
-    implementation('org.postgresql:postgresql')
     implementation 'org.liquibase:liquibase-core'
+    implementation 'org.mongodb:mongo-java-driver'
+    implementation 'org.springframework.data:spring-data-mongodb'
+    implementation 'org.springframework.amqp:spring-rabbit'
+    runtime group: 'org.springframework.cloud', name: 'spring-cloud-dependencies', version: 'Hoxton.SR7', ext: 'pom'
+    //compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-feign', version: '1.4.7.RELEASE'
+    compile group: 'io.github.openfeign', name: 'feign-okhttp', version: '11.0'
```

Comment:
I think if you had commented dependency you then you would not need these two feign-okhttp and feign-gson